.github/workflows/build.yml

```diff
@@ -279,19 +284,19 @@ jobs:
           if-no-files-found: error
 
       - name: Deploy to Play Store
-        if: ${{ matrix.variant == 'prod' && (( github.ref == 'refs/heads/main'
-          && needs.setup.outputs.rc_branch_exists == 0
-          && needs.setup.outputs.hotfix_branch_exists == 0)
-          || (github.ref == 'refs/heads/rc' && needs.setup.outputs.hotfix_branch_exists == 0)
-          || github.ref == 'refs/heads/hotfix-rc' ) }}
+        if: ${{ matrix.variant == 'prod' }}
+        # if: ${{ matrix.variant == 'prod' && (( github.ref == 'refs/heads/main'
+        #   && needs.setup.outputs.rc_branch_exists == 0
+        #   && needs.setup.outputs.hotfix_branch_exists == 0)
+        #   || (github.ref == 'refs/heads/rc' && needs.setup.outputs.hotfix_branch_exists == 0)
+        #   || github.ref == 'refs/heads/hotfix-rc' ) }}
```

Comment:
🤔 Should this be still commented? Or is it commented to check whether this works on `main`?